ui/views.py

```diff
@@ -172,7 +172,9 @@ def get_vocabularies(facets):
         vocab = (vocabulary_id, vocabs[int(vocabulary_id)])
         vocabularies[vocab] = []
         for t_id, count in term_data:
```

Comment:
If we need to wait for the elasticsearch DSL work to update the index then we should hold off on this PR until that work is done.